schemas/elf.yml

```diff
@@ -65,9 +65,12 @@
     - name: go_import_hash
       short: A hash of the Go language imports in an ELF file.
       description: >
-        A hash of the Go language imports in an ELF file. An import hash can be used to
-        fingerprint binaries even after recompilation or other code-level
-        transformations have occurred, which would change more traditional hash values.
+        A hash of the Go language imports in an ELF file excluding standard library imports.
+        An import hash can be used to fingerprint binaries even after recompilation or other
+        code-level transformations have occurred, which would change more traditional hash values.
+
+        The algorithm used to calculate the Go symbol hash and a reference implementation
```

Comment:
Does ECS expect this calculated with or without the inclusion of the stdlib imports?

Reply:
I guess that depends on what we decide here since it's a new symbol. I excluded stdlib packages because it reduces the size of indexed documents. However, I can see _some_ merit in including stdlib given that it is entirely feasible to write malicious code using only stdlib packages. The argument against that is that the packages that you might use to do so are entirely valid for benign applications.

We have to make a decision and this seems like the better of the two.

Reply:
> excluding standard library import

Oh, it is specified. I came up with the question after reading the changes in toutoumomoma and didn't re-read this part.

By including stdlib we would have some signal to use when the binary does not utilize external packages. I assume that for any binary that only uses stdlib that the result is `MD5("")`.

Reply:
> I assume that for any binary that only uses stdlib that the result is `MD5("")`.

Yes, this is correct `d41d8cd98f00b204e9800998ecf8427e`